qa/tasks/cephfs/kernel_mount.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, ctx, test_dir, client_id, client_remote,
             cephfs_name=cephfs_name, cephfs_mntpt=cephfs_mntpt, brxnet=brxnet)
 
         self.rbytes = config.get('rbytes', False)
+        self.snapdirname = config.get('snapdirname', '.snap')
```

Comment:
Please save the `config` parameter to a local class member and name it to `client_config`.